package.json

```diff
@@ -93,6 +93,17 @@
     "url": "https://github.com/perxhealth/react-ui.git"
   },
   "dependencies": {
-    "react-icons": "^4.6.0"
+    "@babel/plugin-syntax-flow": "^7.18.6",
+    "@babel/plugin-transform-react-jsx": "^7.19.0",
+    "@chakra-ui/system": "^2.3.0",
+    "@chakra-ui/toast": "^4.0.0",
+    "@reach/alert": "^0.18.0",
+    "@storybook/api": "^6.5.13",
+    "@storybook/components": "^6.5.13",
+    "@types/node": "^18.11.5",
+    "react-icons": "^4.6.0",
+    "regenerator-runtime": "^0.13.10",
+    "require-from-string": "^2.0.2",
+    "webpack": "^5.74.0"
```

Comment:
@jmhar3 hmm, not super sure what's going on here, did some changes in another branch leak in? I think we'll want to revert every change made here in `package.json`

it may be simpler to pull down canon and start a fresh branch